tests/simulator/test_initialization.py

```
@@ -1,11 +1,8 @@
-from pyphare.cpp import cpp_lib
-
-cpp = cpp_lib()
-
 import unittest
 
 import numpy as np
 from ddt import ddt
+from pyphare import cpp
```

Ruff (0.8.2)

`pyphare.cpp` imported but unused

Remove unused import: `pyphare.cpp`

(F401)